utilities/blob_db/blob_db_impl.cc

```diff
@@ -1732,7 +1732,10 @@ std::pair<bool, int64_t> BlobDBImpl::DeleteObsoleteFiles(bool aborted) {
 
   // directory change. Fsync
   if (file_deleted) {
-    dir_ent_->Fsync();
+    Status s = dir_ent_->Fsync();
+    if (!s.ok()) {
+      return std::make_pair(false, -1);
```

Comment:
I would like to continue execution but print an error log here. BlobDB currently don't expose background error like core rocksdb does.

Reply:
Done.